README.md

```diff
@@ -68,10 +68,22 @@ The default value is `1`.
 
 ## Set GitHub Token
 
-```
+## Set GitHub Token
```

Comment:
Note: this typo that I made was fixed in abe8c1f24607b865460aabd6b20a21a1cd0b7e23